engine/src/flutter/impeller/display_list/paint.cc

```diff
@@ -34,6 +34,31 @@ using DlRect = flutter::DlRect;
 using DlIRect = flutter::DlIRect;
 using DlPath = flutter::DlPath;
 
+void Paint::ConvertStops(const flutter::DlGradientColorSourceBase* gradient,
```

Comment:
We should really do the stop conversion when we record the gradient instead of on each dispatch.

Reply:
https://github.com/flutter/flutter/issues/165409

Reply:
Yeah that SG. We already copy from dart:ui into display list, so the conversion can be done there. Technically this is the same conversion skia does when we dispatch to it, so there should be no change in behavior.